configs.js

```diff
@@ -79,6 +79,11 @@ let configs = (function () {
   const roleID = '1050921202853617724'; // role id to ping, can be obtained by right clicking on the role (ensure to have developer mode on)
   const content = 'Stream is going on break! {role}'; // message to send
 
+  // work/play category options
+  const workCategory = null; // e.g. "Just Chatting"
+  const playCategory = null; // e.g. "Hearthstone"
```

Comment:
Recommend using empty strings so it's more user friendly to none technical users. I can envision some people entering the categories without the quotes, even though there is an example next to it